sklearn/impute/_iterative.py

```diff
@@ -569,9 +575,12 @@ def fit_transform(self, X, y=None):
         self._max_value = np.inf if self.max_value is None else self.max_value
 
         self.initial_imputer_ = None
-        super()._fit_indicator(X)
-        X_indicator = super()._transform_indicator(X)
-        X, Xt, mask_missing_values = self._initial_imputation(X)
+
+        X, Xt, mask_missing_values, missing_mask = self._initial_imputation(X)
```

Comment:
This is not obvious was is the difference between in terms of naming between `mask_missing_values` and `missing_mask`